### PR TITLE
Fix the Error of Korean language translation

### PR DESCRIPTION
**Pull request by jbkim**
I made a fixing the error of Korean language translation.

### CODE REVIEW

File: resources/i18n/ko_KR/uranium.po [hunk 1]
```diff
@@ -136,13 +135,13 @@ msgstr "파일이 이미 있음"
 #, python-brace-format
 msgctxt "@label Don't translate the XML tag <filename>!"
 msgid "The file <filename>{0}</filename> already exists. Are you sure you want to overwrite it?"
-msgstr "파일이 이미 있습니다. 덮어 쓰시겠습니까?"
+msgstr "<filename>{0}</filename> 파일이 이미 있습니다. 덮어 쓰시겠습니까?"
```
Comment (Ghostkeeper) [inline resources/i18n/ko_KR/uranium.po]: Thank you! We had seen that these were missing the `{0}` and such but we didn't have time to tell our translators this and make another translation round before the release. But you fixed it! That's very nice!

Comment (jbkim) [reply resources/i18n/ko_KR/uranium.po]: Ok, I'll try it to 3.3 branch.
Anyway, do you have Korean translator? Wow.

Comment (Ghostkeeper) [reply resources/i18n/ko_KR/uranium.po]: Yeah we hire a translation bureau (Bothof) that does most of our translations.

**Comment by jbkim** [comment]
I fixed the conflicts.

**Comment by Ghostkeeper** [comment]
It seems that after fixing the conflicts, there are no longer any changes left. Did another PR make the same changes? I've lost the overview a bit.

In any case, this PR can be closed because it doesn't change anything any more.

**Pull request closed without merging**